js/repl/ReplOptions.tsx

```diff
@@ -639,6 +639,26 @@ class ExpandedContainer extends Component<Props, State> {
                   type="checkbox"
                 />
               </label>
+              <label className={styles.envPresetRow}>
+                <LinkToDocs className={`${styles.envPresetLabel} ${styles.highlight}`}
+                  section="modules"
+                >
+                  Modules
+                </LinkToDocs>
+                <select
+                  value={envConfig.modules}
+                  className={styles.envPresetSelect}
+                  onChange={this._onEnvPresetSettingChange("modules")}
+                  disabled={!envConfig.isEnvPresetEnabled}
+                >
+                  <option value="auto">auto</option>
```

Comment:
> As a follow up, I'd like to make ESM imports of URLs work in the REPL, so that people can load external deps.

Do you mean allow the import to execute?

Reply:
Yes, when you enable `evaluate` in the sidebar.

Reply:
Something like this: https://github.com/bloomberg/record-tuple-polyfill/pull/30

Reply:
Looks great, I tried doing this with rollup, but only partially.